client/css/style.css

```diff
@@ -1,24 +1,25 @@
-*{
```

Comment:
*, body, html should be handled by proxy.. dont worry these are already being done for you. if you want font size or something different i sugges moving it to your main module wrapper (i called it reviewsMain)